tutorial/6. gpu_solver.ipynb

```diff
@@ -0,0 +1,231 @@
+    "conda activate musica_gpu\n",
+    "pip install --upgrade setuptools pip wheel\n",
+    "pip install nvidia-pyindex\n",
+    "conda install ipykernel scikit-learn seaborn\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2cd76107",
+   "metadata": {},
+   "source": [
+    "## 2. Importing MUSICA\n",
+    "\n",
+    "Importing MUSICA is largerly the same, but with an additional <i>is_cuda_available()</i> function to verify that the GPU is running properly:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0fe92903",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import musica\n",
+    "import musica.mechanism_configuration as mc\n",
+    "import matplotlib.pyplot as plt\n",
+    "from scipy.stats import qmc\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import seaborn as sns\n",
+    "from musica.cuda import is_cuda_available"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "27039537",
+   "metadata": {},
+   "source": [
+    "As with creating the music_box environment in MusicBox's Basic Workflow tutorial, this cell may be slow to run the first time."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d82609a",
+   "metadata": {},
+   "source": [
+    "## 3. Running a Basic Solver on GPU\n",
+    "\n",
+    "This code is a copy of the [Hypercube Tutorial](1.%20basic_workflow.ipynb), but with an if statement added outside the main code to verify that it is running on a GPU.</br>\n",
+    "If you are seeing \"Error: No GPU Available\" being printed, that means a GPU was not detected; verify that your environment has a GPU."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "54dc9c1b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if is_cuda_available():\n",
+    "    A = mc.Species(name=\"A\")\n",
+    "    B = mc.Species(name=\"B\")\n",
+    "    C = mc.Species(name=\"C\")\n",
+    "    species = [A, B, C]\n",
+    "    gas = mc.Phase(name=\"gas\", species=species)\n",
+    "\n",
+    "    r1 = mc.Arrhenius(\n",
+    "        name=\"A_to_B\",\n",
+    "        A=4.0e-3,  # Pre-exponential factor\n",
+    "        C=50,      # Activation energy (units assumed to be K)\n",
+    "        reactants=[A],\n",
+    "        products=[B],\n",
+    "        gas_phase=gas\n",
+    "    )\n",
+    "\n",
+    "    r2 = mc.Arrhenius(\n",
+    "        name=\"B_to_C\",\n",
+    "        A=4.0e-3,\n",
+    "        C=50,\n",
+    "        reactants=[B],\n",
+    "        products=[C],\n",
+    "        gas_phase=gas\n",
+    "    )\n",
+    "\n",
+    "    mechanism = mc.Mechanism(\n",
+    "        name=\"musica_micm_example\",\n",
+    "        species=species,\n",
+    "        phases=[gas],\n",
+    "        reactions=[r1, r2]\n",
+    "    )\n",
+    "\n",
+    "    solver = musica.MICM(mechanism = mechanism, solver_type = musica.SolverType.rosenbrock_standard_order)\n",
```

Comment:
```suggestion
    "    solver = musica.MICM(mechanism = mechanism, solver_type = musica.SolverType.cuda_rosenbrock)\n",
```

It should be this

Reply:
Thanks @K20shores  for catching this. @aidan-winney can you confirm that switching to `cuda_rosenbrock` still works in your case?